guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc

```diff
@@ -253,8 +253,22 @@ Configure Red Hat Subscription Manager to use {ProjectServer} for registration a
 
 .Procedure
 
-To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, complete the following steps:
+To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, use one of the following procedures:
 
+==== Global Registration
+. Generate registration command
++
+UI: Go to Hosts > Register Host, select activation key(s) and generate registration command.
+Or you can generate command via API:
```

Comment:
I like the hammer example, but this says via the API. Could we additionally add an example below hammer for API like so:

```
curl https://username:password@$HOSTNAME/api/registration_commands -X POST -H 'Content-Type: application/json' -d '{\"activation_key\":\"your-key\",\"organization_id\":\"organization_id\"}'"
```